packages/commons/src/pdf-generator/pdfGenerator.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable functional/no-let */
+import path from "path";
+import { fileURLToPath } from "url";
+import { pdfGenerationError } from "pagopa-interop-models";
+import puppeteer, { Browser } from "puppeteer";
+import { buildHTMLTemplateService } from "../index.js";
+
+export interface PDFGenerator {
+  generate: (
+    templatePath: string,
+    context: Record<string, unknown>
```

Comment:
```suggestion
    context: Record<string, string>
```
Since we only have strings in `pdfPayload`, could it be typed better like this?